service/src/worker.rs

```diff
@@ -44,38 +44,29 @@ impl<Config, NodeApi, Enclave, WorkerApiDirect> Worker<Config, NodeApi, Enclave,
 }
 
 #[async_trait]
-pub trait WorkerT {
-	/// Gossip Sidechain blocks to peers.
+/// Gossip Sidechain blocks to peers.
+pub trait AsyncBlockGossiper {
```

Comment:
Removed peers() from this trait and renamed accordingly. 
Idea behind that: We later on don't want to update our peers for every gossiping (i.e. keep ws connection to peers), so I introduced the peers as a worker variable instead. Currently, it's only a vec of urls, but it might be changed to a vec of clients with etablished ws connections.